frontend/packages/dev-console/src/components/import/advanced/DeploymentConfigSection.tsx

```diff
@@ -14,8 +16,13 @@ const DeploymentConfigSection: React.FC<DeploymentConfigSectionProps> = ({ names
     },
   };
 
+  const [field] = useField('resources');
+
   return (
-    <FormSection title="Deployment Configuration" fullWidth>
+    <FormSection
+      title={field.value === Resources.Kubernetes ? 'Deployment' : 'Deployment Configuration'}
```

Comment:
Good catch finding this case as well! Per last comment, please make this always `Deployment`, no check. Sorry about that.

Reply:
reverted the changes and changed the title to Deployment